src/schema/v2/searchCriteriaLabel.ts

```diff
@@ -267,28 +281,32 @@ function getWaysToBuyLabels(waysToBuy: {
   if (acquireable)
     labels.push({
       name: "Ways to Buy",
-      value: "Buy Now",
+      label: "Buy Now",
+      value: "acquireable",
       field: "acquireable",
     })
 
   if (atAuction)
     labels.push({
       name: "Ways to Buy",
-      value: "Bid",
+      label: "Bid",
+      value: "atAuction",
```

Comment:
**question**: is it normal that a string is returned as `value`, and not a boolean value?

Reply:
For boolean filters, a boolean might make more sense.

Reply:
Hmm 🤔  Can we specify several GraphQL types, which should be returned in `type` [here](https://github.com/artsy/metaphysics/blob/4e59d5bfa35bdb5052216f4094540269908622d2/src/schema/v2/searchCriteriaLabel.ts#L53) (in our case it should be `GraphQLString` and `GraphQLBoolean`)?

Reply:
> I wonder if a union type might be useful here?

I tried using `GraphQLUnionType` and the result is the following query structure. IMHO such a request will not be particularly convenient to use and it is better to use the solution you suggested (when true is returned in string format)

```
labels {
  displayValue
  field
  value {
    ... on BooleanValue {
      stringValue: value
    }
    
    ... on StringValue {
      booleanValue: value
    }
  }
}
```